_src/pldoc/static/sass/_base.scss

```diff
@@ -21,7 +21,3 @@ html, body {
     background: palette("grayscale-cool", "x-light");
     font-size: font-size("base");
 }
-
```

Comment:
NOTE: removing redundant style rule (body of ``.pldoc`` pages already has this).